web3_proxy/src/app/mod.rs

```diff
@@ -79,6 +79,7 @@ struct ResponseCacheKey {
 }
 
 impl ResponseCacheKey {
+    #[instrument(level = "trace")]
```

Comment:
i think putting trace instruments at every function is part of what was making things so slow. so we should definitely benchmark (with ethspam) a build with and a build without and see the impact